source/test/F0.Minesweeper.Logic.Tests/MinefieldFactoryTest.cs

```diff
@@ -12,7 +9,8 @@ public void Create_Null_Throws()
 		{
 			MinefieldFactory minefieldFactoryUnderTest = new();
 			Func<IMinefield> create = () => minefieldFactoryUnderTest.Create(null!);
-			create.Should().ThrowExactly<ArgumentNullException>();
+			create.Should().ThrowExactly<ArgumentNullException>()
+				.And.ParamName.Should().Be("options");
```

Comment:
In case of a renaming of the param the test won't pick this up unless someone manually changes the test. Is this desired by us?
From my side it's not a blocker. Just something that I personally would usually avoid since this doesn't really test functionality.

Reply:
The type and the method is `public` - technically renaming the parameter is a breaking change - but we don't publish an API so I guess it is indeed fine to ignore.

Reply:
Or is there a reflection-y way to `nameof` the first argument of the method?
Or would this be overkill? 😂

Reply:
Discussed: We will remove the assertion of `ParamName` again.